cni/azure-linux.conflist

```diff
@@ -9,6 +9,9 @@
          "ipsToRouteViaHost":["169.254.20.10"],
          "ipam":{
             "type":"azure-vnet-ipam"
+         },
```

Comment:
i dont think its required

Reply:
I think it is, when this comma wasn't there, `network_linux.go` threw an error saying  "NetworkMonitor is not set in conflist"
```
netmon := nwCfg.NetworkMonitor
		if netmon.Name == "" {
			return fmt.Errorf("NetworkMonitor is not set in conflist")
		}
```